jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -0,0 +1,126 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+const PropTypes = require('prop-types');
+const shapeup = require('shapeup');
+const XTerm = require('xterm');
+require('xterm/lib/addons/terminado/terminado');
```

Comment:
Does this add something to the global state of the app? Why isn't it assigned to a variable and used.

Reply:
This is how xterm loads plugins. Basically a side effect of loading the file is changing the prototype of he term object. It's too implicit, but not something we can change. Madison please maybe add a comment, so that this behavior is not confusing for future readers?